examples/use_openai_assistant_azure.rs

```diff
@@ -0,0 +1,100 @@
+use std::ffi::OsStr;
```

Comment:
This file is an example of how to use Assistants via Azure OpenAI. 

It is mostly a copy of `use_openai_assistant`. The main change is that when creating `OpenAIFile`, `OpenAIVectorStore` or `OpenAIAssistant` you need to set the version to `OpenAIAssistantVersion::Azure`.

It also requires env variables `OPENAI_API_URL` and `OPENAI_API_KEY` to be set to the corresponding Azure values.

Reply:
fantastic!!